src/status_im2/contexts/quo_preview/drawers/permission_drawers.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.quo-preview.drawers.permission-drawers
+  (:require [quo2.core :as quo]
```

Comment:
this component was missing a preview. Discussed with @cammellos as it is unclear of what this component should be more so than a styled container. 
I included some examples so we can see how to get the designs from this at the very least. 👍